test/integration_test.dart

```diff
@@ -366,6 +378,8 @@ Future<void> main() async {
           ),
         ));
 
+    Utils.feedbackEnabled = true;
```

Comment:
Since I said to remove this from Utils, what you can do to test it without messing up the code is add a `static Map<String, dynamic> overrides` field to `RemoteConfigService` that normally in the app would be null, but in the test you can set to {_feedbackEnabled: true}. Then in the `feedbackEnabled` getter you do something like `overrides?.get(_feedbackEnabled) ?? _remoteConfig...`.